components/base/CollectionSelect.vue

```diff
@@ -5,36 +5,48 @@
         {{ $t('createNftExplainer') }}
       </small>
     </div>
-    <NeoField
-      :label="$t('collection')"
-      :message="$t('Select collection where do you want mint your token')">
-      <b-select
+    <NeoField :message="$t('selectCollectionDescription')">
+      <NeoSelect
         v-model="selectedCollection"
-        placeholder="Select a collection"
-        expanded>
-        <option disabled selected value="">--</option>
-        <option v-for="option in collections" :key="option.id" :value="option">
-          {{ option.name || option.id }} ({{ option.totalCount }})
-        </option>
-      </b-select>
+        :label="$t('collection')"
+        :placeholder="$t('selectCollection')"
+        :options="selectOptions"
+        expanded />
     </NeoField>
```

Comment:
Please replace this by:
```ts
    <NeoField>
      <template #label>
        <div>{{ $t('collection') }}</div>
        <div class="has-text-weight-light is-size-7 mb-3">
          {{ $t('selectCollectionDescription') }}
        </div>
      </template>
      <NeoSelect
        v-model="selectedCollection"
        :placeholder="$t('selectCollection')"
        :options="selectOptions"
        expanded />
    </NeoField>
```

Reply:
<img width="704" alt="image" src="https://github.com/kodadot/nft-gallery/assets/64609060/254b6bda-107c-4827-be46-e7760aa90273">

This is the output now